### DISCUSSION TITLE
Use specific commit or branch in package.json

### DISCUSSION THREAD

**Discussion by angel-garcia-lopez** [ionic-team/capacitor#3798] [Q&A]
Hello,

Since #3638 has been fixed but is not released in npm yet in current 2.4.2, I am looking for a way to use the "2.x" branch or a specific commit like ce6d9423842dbf6dc889dff1c11523181e6b9b6b in the "package.json > dependencies" until the next release.

My idea is just to perform a npm install and get it working.

Things I've tried and didn't work:
- "@capacitor/core": "git://github.com/ionic-team/capacitor.git#1c47e15fb01cd5e730ce5bcd7dddf436fd874673",
- git+https://..... (this works for some cordova plugin forks)
- Using the .zip link that github generates for a commit or branch if requested

Thanks,

**Comment by jcesarmobile** [comment]
Since Capacitor repository is a monorepo, it's not possible to install the packages from github directly.

If you want to use a development version you'll have to clone the repo, checkout that branch and install from a local folder like `npm install /local/path/to/capacitor/android/`

**Reply by thor9n** [reply]
I've managed to add the library but Capacitor doesn't interpret it as a plugin and skips it. How can I make sure that `ionic cap sync` interprets it as a plugin and installs it?
